back/spec/services/xlsx_service_spec.rb

```diff
@@ -122,7 +122,7 @@ def xlsx_to_array(xlsx, sheet_index: 0)
     it 'contains extra columns for custom user fields' do
       create(:custom_field_domicile)
       area = create(:area, title_multiloc: { 'en' => 'Center' })
-      users.first.update!(custom_field_values: { 'domicile' => area.id })
+      users.first.update(custom_field_values: { 'domicile' => area.id })
```

Comment:
Unrelated change. I noticed I had used an exclam in other work, that is not needed.